developers/in-depth-tutorial-insurance-claims-arbitration.md

```diff
@@ -0,0 +1,387 @@
+# In depth tutorial: Insurance claims arbitration
+
+This section covers [Insurance claims arbitration contract](https://github.com/UMAprotocol/dev-quickstart/blob/main/contracts/InsuranceArbitrator.sol), which is available at the [developer's quick-start repo](https://github.com/UMAprotocol/dev-quickstart). This tutorial shows example on how insurance claims can be resolved and settled through [Optimistic Oracle V2](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/oracle/implementation/OptimisticOracleV2.sol).
+
+You will find out how to test and deploy this smart contract and how it integrates with Optimistic Oracle. Please see more details on [how does UMA's Oracle work](../protocol-overview/how-does-umas-oracle-work.md) section.
+
+### Insurance Arbitrator Contract
+
+This smart contract allows insurers to issue insurance policies by depositing insured amount, designating the insured beneficiary and describing insured event.
+
+At any time anyone can submit a claim that the insured event has occurred by posting oracle bonding. Insurance Arbitrator resolves the claim through Optimistic Oracle by passing a templated question with insured event description in ancillary data using `YES_OR_NO_QUERY` price identifier as specified in [UMIP-107](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-107.md).
+
+If the claim is confirmed and settled through Optimistic Oracle this contract automatically pays out insurance coverage to the insured beneficiary. If the claim is rejected the policy continues to be active ready for the subsequent claim attempts.
+
+### Development environment and tests
```

Comment:
we should show how to do this via remix as well! I can do this via a subsequent PR